lib/types/user.ts

```diff
@@ -1,7 +1,14 @@
 export type UserType = 'TALENT' | 'EMPLOYER' | 'AGENCY';
 
+export interface FullName {
+  firstName?: string;
+  middleName?: string;
+  lastName: string;
+}
+
```

Comment:
Maybe we should put interfaces that are used by several other interfaces (e.g. FullName will be used by talent, user and employer) to a file with "common" types.